lib/vanagon/engine/pooler.rb

```diff
@@ -7,7 +7,7 @@ class Pooler < Base
       attr_reader :token
 
       # The vmpooler_template is required to use the pooler engine
-      def initialize(platform, target = nil)
+      def initialize(platform, target = nil, opts = {})
```

Comment:
Since Vanagon depends on Ruby 2.1 or newer, it's safe to use keyword arguments. Would that be easier than continuing to use the old Options Hash pattern?